tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/ValidateAsyncV5Integration.cs

```diff
@@ -0,0 +1,63 @@
+﻿// <copyright file="ValidateAsyncV5Integration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL;
+
+/// <summary>
+/// GraphQL.Validation.DocumentValidator calltarget instrumentation for GraphQL 4
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = GraphQLCommon.GraphQLAssembly,
+    TypeName = "GraphQL.Validation.DocumentValidator",
+    MethodName = "ValidateAsyncCoreAsync",
+    ReturnTypeName = ClrNames.GenericParameterTask,
+    ParameterTypeNames = new[] { "GraphQL.Validation.ValidationContext", "System.Collections.Generic.IEnumerable`1[GraphQL.Validation.IValidationRule]" },
+    MinimumVersion = GraphQLCommon.Major5,
+    MaximumVersion = GraphQLCommon.Major5,
+    IntegrationName = GraphQLCommon.IntegrationName)]
+[Browsable(false)]
+[EditorBrowsable(EditorBrowsableState.Never)]
+public class ValidateAsyncV5Integration
+{
+    private const string ErrorType = "GraphQL.Validation.ValidationError";
```

Comment:
nit - you repeat this one, whereas you have made internal the ErrorType for execution. 
We could maybe move them to `GraphQLCommon`